### PR TITLE
1.0/183 cookie banner

### LINKED ISSUE #183: Cookie Banner
## Description

We need to give people the option to opt into having Google Analytics track them on the site - check re: cookie tracking widget

## Plan

- [ ] Think about how we do this
- [ ] What copy are we using?
- [ ]  Accessibility thought!
- [ ] Do we record things about the user between visits?
- [ ] Implement above if required
- [ ] Look into adding cookie tracking widget (https://www.cookiehub.com/)

##  Acceptance Criteria

- [ ] People opt-in not opt-out of cookies
- [ ] The cookie notice is clear and easy to find
- [ ] Opt in and opt out are given equal weight
- [ ] People can change their minds

### PR DESCRIPTION
**Pull request by Nikomus**
Fixes #183 

## Description

- Adds a banner with some styling in the layout
- The buttons update a consent flag on the model
- Needs to check the flag before confirming GA events

@neontribe/contemplating-action

### CODE REVIEW

**Review by katjam — APPROVED** [review]
This looks great. :open_umbrella:

File: src/Copy/Keys.elm [hunk 1]
```diff
@@ -43,6 +43,13 @@ type Key
     | ContentLinkLong
     | ContentLinkMedium
     | ContentLinkShort
+    | ContactLinkLong
```
Comment (katjam) [inline src/Copy/Keys.elm]: These contact ones have gone.

File: src/styles/layout.scss [hunk 2]
```diff
@@ -638,6 +638,32 @@ nav,
   }
 }
 
+/* ------------------------------
```
Comment (katjam) [inline src/styles/layout.scss:641]: Assuming this is good for all viewport sizes?

**Review by katjam — CHANGES_REQUESTED** [review]
Looks reasonable but you need to get rid of the contact keys.

Also thoughts aound if 0 and 2 mean the same thing - can we simplify?

**Comment by Nikomus** [comment]
@katjam I think I need three states for showing the banner (undecided), confirmed yes and confirmed no. I could have a pair of bools for clicked/not clicked and yes/no.

**Comment by katjam** [comment]
@Nikomus But you only make decisions on 1 or _

Unless I missed something.

File: src/Copy/Keys.elm (hunk 1 above)
Comment (Nikomus) [reply src/Copy/Keys.elm]: Weird merge, release put them back.. 🤔

**Comment by katjam** [comment]
I see you have 1 _ and 0 _ maybe you need all 3. Ignore me.

**Comment by Nikomus** [comment]
The other decision is on 0 or _ in Cookies.elm to display or not.

**Review by katjam — APPROVED** [review]
Do it. Lovely job. :lion:

**Pull request merged**